client/my-sites/backup/status/index.jsx

```diff
@@ -34,10 +34,9 @@ export const DailyStatus = ( { selectedDate } ) => {
 	useDailyBackupStatus( siteId, moment( selectedDate ).subtract( 1, 'day' ) );
 	useDailyBackupStatus( siteId, moment( selectedDate ).add( 1, 'day' ) );
 
-	const lastBackupDate = useDateWithOffset(
-		lastBackupBeforeDate?.activityTs,
-		!! lastBackupBeforeDate
-	);
+	const lastBackupDate = useDateWithOffset( lastBackupBeforeDate?.activityTs, {
+		shouldExecute: !! lastBackupBeforeDate,
+	} );
```

Comment:
Now `useDateWithOffset` accepts a config object so we need to pass `shouldExecute` inside of one.